### PR TITLE
Enable page_size in new pdc client

### PR DESCRIPTION
**Pull request by bliuredhat**
JIRA: PDC-1158

### CODE REVIEW

**Comment by simozhan** [comment]
it seems the test failed.

**Comment by bliuredhat** [comment]
Hi, all( @simozhan, @lubomir ,@xychu, @ycheng); 'Enable page_size' is not available for all methods, maybe just for "list"method  and when using extract_arguments() function. Can you give me some more perfect ways to edit this patch?  I think that we don't need to set page_size for ["create" , "update"] methods.
The position of page_size in pdc command as followed:
$   pdc  [-s server]  [page_size   NUM]  command

**Comment by lubomir** [comment]
Alternative way would be to do this change in `get_paged` function. It would probably need to move to `PDCClient` class. I think most of the paged requests go through that.

**Comment by xychu** [comment]
I agree with @lubomir.
It makes more sense that `page_size` get handled and passed to inside functions in `get_paged`.

**Comment by bliuredhat** [comment]
@lubomir  and @xychu , Thanks and I have done it.

**Comment by lubomir** [comment]
Would the type conversion be easier if done with argument parser?
https://docs.python.org/2/library/argparse.html#type

**Comment by simozhan** [comment]
As you move get_paged() to PDCClient, i think you need to update README.markdown as well.

**Comment by lubomir** [comment]
The old function should actually not be deleted: the docstring should be updated to mark it as deprecated. It should only be removed in next release.

**Comment by bliuredhat** [comment]
@lubomir ; yes , your ideal is good; do the command like:
 $ pdc  [-s server]  command   --page-size NUM
?

**Comment by lubomir** [comment]
@bliuredhat The `--page-size` argument should be defined only once, so it would have to come before the command: `pdc -s prod --page-size 11 some-command`.

**Comment by bliuredhat** [comment]
@lubomir , yes~, the command work as you show!

**Comment by xychu** [comment]
I think [ArgumentParser type](https://docs.python.org/2/library/argparse.html#type) is worth to try :-)

**Comment by bliuredhat** [comment]
@xychu , yes; I think it's available ~!

File: pdc_client/__init__.py [hunk 1]
```diff
@@ -42,19 +42,8 @@ def read_config_file(server_alias):
...
-    :param res:     what resource to connect to
-    :param kwargs:  filters to be used
-
-    ::
-
-        # Example: Iterate over all active releases
-        for release in get_paged(client['releases']._, active=True):
-            ...
+    Warning: This method have been removed to PDCClient class;
+    and then will be deleted in next release.
```
Comment (xychu) [inline pdc_client/__init__.py]: The docstring will not be rendered, so maybe we could add one [PendingDeprecationWarning](https://docs.python.org/2.7/library/exceptions.html#exceptions.PendingDeprecationWarning) as well.

**Pull request merged**